Documentation/RepoApi.md

```diff
@@ -28,25 +28,23 @@ There will be a number of actions source-build will need the repos to implement.
 
 1. `Build` - Assemble the product from the current source contained in the repo.
 
-source-build formats arguments using a style that is compatible with MSBuild. The reasoning for this is that the majority of repos have an MSBuild based build system. Using this format will allow for arguments to easily be passed through to MSBuild. Similarly, both action and parameter arguments are case-insensitive, just like MSBuild default behavior.
-
-Actions will take the form of `-t:ActionName`, just like MSBuild targets are passed to `msbuild.exe`.
+Actions will take the form of `ActionName` and are the first parameter passed to the build script. Action names are case insensitive.
```

Comment:
I think we should add that an actions won't start with `-` so that repos can detect that and have a default behavior for their "clone and type `build`" dev scenario.